src/skmultiflow/data/stream.py

```diff
@@ -0,0 +1,30 @@
+import pandas as pd
+from tornado import gen
+from streamz import Stream, Source
+from skmultiflow.data.base_generator import BaseGenerator
+
+@Stream.register_api(staticmethod)
+class from_generator(Source):
```

Comment:
Same comment about naming as in the `BaseGenerator` class.

Reply:
`from_multiflow_generator` ?